### DISCUSSION TITLE
Disable Bluetooth on NV14 and EL18

### DISCUSSION THREAD

**Discussion by ishampadron** [EdgeTX/edgetx#4591] [Ideas]
Add the ability to toggle off BT on the NV14 and EL18

Context: In today's flying environment with modern devices almost every one is broadcasting BT or WIFI. Their goggles , controllers and FCs to name a few. I would like to be able to turn it off to not interfere with WIFI timing systems and to waste battery unnecessarily. As far as I know it's not really implemented on the NV14 and EL18.

**Comment by pfeerick** [comment]
Are you sure it's even turned on? There is code to turn it on and off in the board file for the NV14 (and EL18), but since bluetooth is not enabled for either by default, it is most likely never powered up in the first place. 

https://github.com/EdgeTX/edgetx/blob/6b6a12ab45474929178a3162523bcba21b9fe509/radio/src/targets/nv14/board.h#L101-L102
https://github.com/EdgeTX/edgetx/blob/6b6a12ab45474929178a3162523bcba21b9fe509/radio/src/targets/pl18/board.h#L72-L73

**Reply by ishampadron** [reply]
When the transmitter is on I can see Nirvana on my phone under by settings. I can try to connect to it and it will ask me for a password but 1234 and 0000 both fail. Both the EL18 and NV14 broadcast "Nirvana"
